### ISSUE TITLE
Invalid escape sequence

### ISSUE THREAD

**Issue by rolandmoldovan** [ElrondNetwork/elrond-go#639]
Node fails to start with the following error message when NodeDisplayName  = "\a\b\c"

```
C:\Users\Self\go\src\github.com\ElrondNetwork\elrond-go\cmd\node\config\prefs.toml
(3, 27): invalid escape sequence: \a
```

**Comment by bogdan-rosianu** [comment]
Hello. This is caused when the TOML file is parsed. However, if you still want backslashes in your name, you can use `NodeDisplayName = "\\a\\b\\c"` which will result in `\a\b\c` name

**Comment by iulianpascalau** [comment]
Fixed, reference PR #1513
